cmd/osv-scanner/main_test.go

```diff
@@ -503,13 +503,27 @@ func TestRun_LocalDatabases(t *testing.T) {
 			exit: 0,
 		},
 	}
+
+	var testDir string
+	var cleanupTestDir func()
+	if os.Getenv("TEST_ACCEPTANCE") != "true" {
```

Comment:
we should be able to use `cleanupTestDir`, and we also need to call `defer cleanupTestDir()` to ensure it gets cleaned up